store/src/java/com/zimbra/cs/service/util/VolumeConfigUtil.java

```diff
@@ -93,7 +96,7 @@ public static void validateCreateVolumeRequest(CreateVolumeRequest request, Volu
 
         // validate current
         if (false != volInfoRequest.isCurrent() &&
```

Comment:
why are we checking `volInfoRequest.isCurrent()` twice in this condition?

Reply:
@inpati this condition will never be satisfied. can you please check this again?

Reply:
Done. fixed. Basically we are checking whether if current is not true and not false, this is corner case as anybody can pass string in this current variable.